src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/DocumentDetails.scala

```diff
@@ -0,0 +1,7 @@
+package it.pagopa.interop.backendforfrontend.api.impl
+
+import akka.http.scaladsl.model.ContentType
+
+import java.io.ByteArrayOutputStream
+
+final case class DocumentDetails(name: String, contentType: ContentType, data: ByteArrayOutputStream)
```

Comment:
this class can be avoided, you can just pass the parameters to the function

Reply:
+1